data-collection/builder/localize-your-app/index.md

```diff
@@ -0,0 +1,29 @@
+---
+sidebar_position: 10
+title: Localize your app
+---
+
+Huma app builder has a **Hybrid** approach to localization and translation.
+
+1. **Huma-managed texts**: Some texts, such as licensed questionnaires, static texts within the app, and more, are
+   handled by Huma. Huma will take care of their translation.
+2. **Widget-specific texts**: You are responsible for translating the texts you configure in your widgets via the
+   localization section in the settings.
+
+    - Any text you configure in your panel like the title and description here:
+
+      ![alt text](<../assets/localization-builder-config.png>)
+    - Will appear in the localization settings of your app. Go to the settings and switch to the localization tab:
+
+      ![alt text](<../assets/localization-settings.png>)
+    - Add a new language you want to add support for
+
+      ![alt text](<../assets/localization-new-lang.png>)
+    - Start translating, Then if your user has their selected language (OS language) set in the selected language they
+      will see your localization instead:
+
+      ![alt text](<../assets/localization-localize.png>)
+    - Once done click save, and preview the changes in the app.
```

Comment:
@amirh-huma do you want to explain a bit more about import export as well? because it was a bit complicated so they should know not to change the structure, also I would assume they would use that often.

Thanks